functions/Show-DbaGitHubDemo.ps1

```diff
@@ -0,0 +1,80 @@
+Function Show-DbaGitHubDemo
+{
+<#
+.SYNOPSIS 
+Simple template
+
+.DESCRIPTION
+By default, all SQL Agent categories for Jobs, Operators and Alerts are copied.  
+
+.PARAMETER SqlServer
+The SQL Server instance.You must have sysadmin access and server version must be SQL Server version 2000 or higher.
+
+.PARAMETER SqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. 
+
+.NOTES 
+Original Author: You (@YourTwitter, Yourblog.net)
```

Comment:
Thanks so much for contributing a new command! You forgot to update yourself as the original author.